src/main/java/net/tropicraft/core/common/block/TropicraftBlocks.java

```diff
@@ -127,6 +127,7 @@
 
     public static final Block BAMBOO_CHEST = new BambooChestBlock(Block.Properties.create(Material.BAMBOO).sound(SoundType.BAMBOO));
     public static final Block SIFTER = new SifterBlock(Block.Properties.create(Material.WOOD));
+    public static final Block VOLCANO = new VolcanoBlock();
```

Comment:
This should follow the pattern of the other blocks and create the `Block.Properties` here